src/Build/mkdocs.jl

```diff
@@ -0,0 +1,94 @@
+function makemkdocs(build)
+    site_name = "Docile.jl"
+    repo_url = "https://github.com/MichaelHatherly/Docile.jl"
+    site_description = "Julia package documentation system."
+    site_author = "Michael Hatherly"
+    theme = "readthedocs"
+    docs_dir = joinpath(splitdir(Base.source_dir())[end], build)
```

Comment:
Should probably look it up from existing yml (depend on YML package) ?

That'll be easier I think.